Makefile

```diff
@@ -115,9 +115,9 @@ endif
 
 run-test:
 	$(info Running unit tests...)
-	hack/make-rules/test.sh $(WHAT) $(TESTS)
+	go test -v -coverprofile cover.out -race -parallel 8  ./pkg/...
```

Comment:
@asm582, @KPostOffice, @dmatch01, @metalcycling let me know if you are OK with the verbose output from testing and with the exclusion of the `./cmd` package.